codebuild/spec/buildspec_omnibus.yml

```diff
@@ -120,43 +120,6 @@ batch:
           S2N_LIBCRYPTO: 'awslc-fips'
           BUILD_S2N: 'true'
 
-    - identifier: s2nAsanOpenSSL111Coverage
```

Comment:
Why not just update these with the new job?

Reply:
Hmm. I guess that depends on how exactly we are using the omnibus job. I usually think of it as "approximate list of the CI jobs that we run", and since the Asan jobs are now documented in the Asan buildspec, I figured it made sense to remove it.

But perhaps your point is that we do run the omnibus job, in which case we should keep the asan jobs in there? I'd be fine with that in the short term to unblock the PR, but long term it feels a bit odd to duplicate all of our CI build specs. Could whatever is using the omnibus just switch the using the normal CI jobs that we have?